app/controllers/users/details.js

```diff
@@ -0,0 +1,57 @@
+import Ember from "ember";
+
+export default Ember.Controller.extend({
+  organisationService: Ember.inject.service(),
+  organisationsUserService: Ember.inject.service(),
+  userOrganisationDetails: Ember.computed("model", function() {
```

Comment:
```suggestion
  userOrganisationDetails: Ember.computed("model", "model.organisationsUsers.[]", "model.organisationsUsers.@each.userStatus", function() {
```

I fear it might not react properly otherwise